app2.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+var storeHourArray = ['','6am','7am','8am','9am','10am','11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm','7pm','8pm','Total'];
+var storeName = ['1st and Pike','SeaTac','Seattle Center','Capitol Hill','Alki'];
+var body = document.getElementsByTagName('body')[0];
+
+function cookiesStore(name,maxcustomer,mincustomer,avgCookieHour) {
+  this.name = name;
+  this.maxcustomer = maxcustomer;
+  this.mincustomer = mincustomer;
+  this.avgCookieHour = avgCookieHour;
+  this.cookieTotal = [];
+  this.getRandomCustomer = function() {
+    return Math.floor(Math.random() * (this.maxcustomer - this.mincustomer + 1)) + this.mincustomer;
+  };
+  console.log('It works so far.');
```

Comment:
console logs are extremely useful for debugging / verifying code is actually doing what you think it should do.   Once code is working remove the console logs from your code.